### PR TITLE
Feature/responsiveness

### PR DESCRIPTION
**Pull request by JeffShepherd**
### What Changed?
- Update header logo
- Optimize database and incidents views for responsiveness
- Remove ContactShowCard and associate code as this is obsolete now
### What's Next?
- Adjust responsiveness of chart and incident info views once they are complete
### Known Issues?
- No major issues, but components will need small tweaks to complete
### Test Coverage?
- N/A

### CODE REVIEW

File: src/API-calls.js [hunk 1]
```diff
@@ -45,11 +45,6 @@ export const runNameQuery = (queryString) => {
     .then(checkResponse)
 }
 
-export const fetchContactDets = (id) => {
-  return fetch(`${baseURL}/contacts/${id}`)
-    .then(checkResponse)
-}
-
 export const fetchIncidentContacts = (id) => {
```
Comment (percworld) [inline src/API-calls.js:48]: Good eye!

File: src/components/App/App.css [hunk 2]
```diff
@@ -1,21 +1,11 @@
...
 .content-wrap {
   padding-bottom: 3.5rem;
-  overflow-y: scroll;
+  /* overflow-y: scroll; */
 }
-
-nav {
-    display: flex;
-    justify-content: center;
-    border-bottom: 1px solid lightgray;
```
Comment (percworld) [inline src/components/App/App.css:17]: I agree, had just changed this color to match our palette

File: src/components/DatabaseMenu/DatabaseMenu.css [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
-  width: 90vw;
+  width: 100%;
+  margin: 0 auto;
+
 }
 
-.database-menu {
-  flex-direction: column;
-  width: 100em;
+@media only screen and (max-width: 768px) {
```
Comment (percworld) [inline src/components/DatabaseMenu/DatabaseMenu.css:11]: This is great, appreciate the look into media queries

File: src/components/DatabaseSearch/DatabaseSearchSlice.js [hunk 4]
```diff
@@ -38,13 +29,6 @@ const slice = createSlice({
         state.status = 'idle';
         state.searchResults = action.payload;
       })
-      .addCase(displayContactDets.pending, (state) => {
```
Comment (percworld) [inline src/components/DatabaseSearch/DatabaseSearchSlice.js:41]: Guess we Don't need this status update anymore.

File: src/components/Header/Header.css [hunk 5]
```diff
@@ -1,20 +1,9 @@
...
-  text-align: center;
-  margin-top: 3vh;
-  margin-left: 1vw;
-  margin-bottom: 0;
-  padding: 0;
-  display: inline-block;
-  position: relative; 
-  color: #23395B;
-  font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif
+  justify-content: space-between;
```
Comment (percworld) [inline src/components/Header/Header.css:3]: Good look on the new header style

**Review by cbmackintosh — APPROVED** [review]

**Pull request merged**